<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QDialog" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>761</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="3">
     <widget class="QLabel" name="NumberLabel">
      <property name="text">
       <string>Number:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="NameLineEdit"/>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="NumberTextEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="AddressLabel">
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="NameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="AddressTextEdit"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="EmailLabel">
      <property name="text">
       <string>Email: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="EmailTextEdit"/>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Sign in</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
